compiler/src/iree/compiler/DispatchCreation/FormDispatchRegions.cpp

```diff
@@ -1065,15 +1065,10 @@ createFusionGroups(TensorDimTrackingRewriter &rewriter,
         }
       }
 
-      if (failed(moveOperandDefs(rewriter, consumer, regionOp, dominanceInfo,
```

Comment:
I moved this into `moveFollowingOpIntoDispatchRegion` to provide broader support in general in different places where it is used. This means that below the error is replaced with continue, altering the functionality here, but CI seems ok with these changes.